src/middleware/authorization.js

```diff
@@ -0,0 +1,16 @@
+/* Determines API permission levels based on user role */
```

Comment:
This file doesn't need any changes (as far as I know), as long as the authentication file is able to place the user object on the `req` object, then this file should just work out of the box.